core-rust/state-manager/src/staging/result.rs

```diff
@@ -215,7 +260,7 @@ impl ProcessedResult {
     }
 }
 
-pub struct ProcessedTransactionCommit {
+pub struct HashStructuresDiff {
```

Comment:
good idea; done 👍 

_regarding the merge: I will wait until https://github.com/radixdlt/babylon-node/pull/367 merges (there will be conflicts and I prefer to resolve them in my small PR rather than in that monster)_